### PR TITLE
update RN to 0.64 

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Comment by brunobar79** [comment]
Before merging this one I'd love for @skylarbarrera  to take a look to make sure we aren't introducing any regressions

File: android/app/build.gradle [hunk 1]
```diff
@@ -254,7 +254,7 @@ android {
             def abi = output.getFilter(OutputFile.ABI)
             if (abi != null) {  // null for the universal-debug, universal-release variants
                 output.versionCodeOverride =
-                        versionCodes.get(abi) * 1000 + defaultConfig.versionCode
+                        defaultConfig.versionCode * 1000 + versionCodes.get(abi)
```
Comment (brunobar79) [inline android/app/build.gradle:257]: What's is this about? Just curious

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -68,7 +68,6 @@
           <data android:scheme="https" android:host="rnbwapp.com" />
         </intent-filter>
       </activity>
-      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
```
Comment (brunobar79) [inline android/app/src/main/AndroidManifest.xml:71]: Can we use the dev menu if we remove this?

**Review by brunobar79 — APPROVED** [review]

File: android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (osdnk) [reply android/app/src/main/AndroidManifest.xml:71]: it is moved to another file

File: android/app/build.gradle (hunk 1 above)
Comment (osdnk) [reply android/app/build.gradle:257]: Idk, it was in diff :)

**Pull request closed without merging**